overlord/snapstate/snapmgr_test.go

```diff
@@ -486,7 +486,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 
 	// check link snap summary
 	linkTask := ta[len(ta)-1]
-	c.Check(linkTask.Summary(), Equals, `Make snap "some-snap" available to the system (rev 11)`)
+	c.Check(linkTask.Summary(), Equals, `Make snap "some-snap" (rev 11) available to the system`)
```

Comment:
Sorry, I thought it was only on the other file that it was off, but it seems to use rev consistently.

It seems slightly nicer to standardize as "... snap %q (%s) ..." as we discussed online. It looks slightly nicer, it's shorter which is good as we'll use it a lot, and people will very quickly learn that this is the revision.